File: ncDados/dEtiquetaProdutoImpressao.vb
Imports ncDados.nsProduto

Namespace nsEtiquetaProdutoImpressao

  Public Class ColecaoEtiquetaProdutoImpressao
    Inherits List(Of dEtiquetaProdutoImpressao)
  End Class

  Public Class dEtiquetaProdutoImpressao

    Private _loja As String
    Private _fabricante As String
    Private _produto As dProduto
    Private _colecaoProdutoItem As ColecaoProdutoItem
    Private _corNome As String
    Private _data As String
    Private _produto_cid As String
    Private _codigoBarras As String

    Public Property loja() As String
      Get
        Return _loja
      End Get
      Set(ByVal value As String)
        _loja = value
      End Set
    End Property

    Public Property fabricante() As String
      Get
        Return _fabricante
      End Get
      Set(ByVal value As String)
        _fabricante = value
      End Set
    End Property

    Public Property produto() As dProduto
      Get
        Return _produto
      End Get
      Set(ByVal value As dProduto)
        _produto = value
      End Set
    End Property

    Public Property colecaoProdutoItem() As ColecaoProdutoItem
      Get
        Return _colecaoProdutoItem
      End Get
      Set(ByVal value As ColecaoProdutoItem)
        _colecaoProdutoItem = value
      End Set
    End Property

    Public Property corNome() As String
      Get
        Return _corNome
      End Get
      Set(ByVal value As String)
        _corNome = value
      End Set
    End Property

    Public Property data() As String
      Get
        Return _data
      End Get
      Set(ByVal value As String)
        _data = value
      End Set
    End Property

    Public Property produto_cid() As String
      Get
        Return _produto_cid
      End Get
      Set(ByVal value As String)
        _produto_cid = value
      End Set
    End Property

    Public Property codigoBarras() As String
      Get
        Return _codigoBarras
      End Get
      Set(ByVal value As String)
        _codigoBarras = value
      End Set
    End Property

  End Class

End Namespace

File: Modelos/dEtiquetaProdutoImpressao.cs
using System;
using System.Collections.Generic;

namespace Modelos
{
    public class ColecaoEtiquetaProdutoImpressao : List<dEtiquetaProdutoImpressao>
    {
    }

    public class dEtiquetaProdutoImpressao
    {
        private string _loja;
        private string _fabricante;
        private dProduto _produto;
        private ColecaoProdutoItem _colecaoProdutoItem;
        private string _corNome;
        private string _data;
        private string _produto_cid;
        private string _codigoBarras;

        public string loja
        {
            get => _loja;
            set => _loja = value;
        }

        public string fabricante
        {
            get => _fabricante;
            set => _fabricante = value;
        }

        public dProduto produto
        {
            get => _produto;
            set => _produto = value;
        }

        public ColecaoProdutoItem colecaoProdutoItem
        {
            get => _colecaoProdutoItem;
            set => _colecaoProdutoItem = value;
        }

        public string corNome
        {
            get => _corNome;
            set => _corNome = value;
        }

        public string data
        {
            get => _data;
            set => _data = value;
        }

        public string produto_cid
        {
            get => _produto_cid;
            set => _produto_cid = value;
        }

        public string codigoBarras
        {
            get => _codigoBarras;
            set => _codigoBarras = value;
        }
    }
                    
}